simpa/core/simulation_modules/reconstruction_module/reconstruction_utils.py

```diff
@@ -403,29 +403,21 @@ def preparing_reconstruction_and_obtaining_reconstruction_settings(
             time_spacing_in_ms, torch_device)
 
 
-def get_reconstruction_processing_unit(global_settings):
+def get_reconstruction_processing_unit(global_settings: Settings) -> torch.device:
```

Comment:
I don't think we still need this function when we have the generalized function. Remove it and use the `get_processing_device` function